SlowCheetah.VisualStudio/SlowCheetahPackage.cs

```diff
@@ -586,6 +586,11 @@ private IEnumerable<string> GetPublishProfileTransforms(IVsHierarchy hierarchy,
         /// <returns>True if the project supports transformation</returns>
         private bool ProjectSupportsTransforms(IVsProject project)
         {
+            if (this.NuGetManager.ProjectSupportsNuget(project as IVsHierarchy))
```

Comment:
Can you manually verify if this works in VS 2013 and then in 2017? Also check if it works for all of the supported project extensions. If so, we can delete the old way of checking.